### PR TITLE
Update UG

### PR DESCRIPTION
**Pull request by harithh07**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -76,32 +80,64 @@ Format: `help`
...
+* Tags must be created before they can be assigned to a person. Refer to [newtag](#creating-a-new-tag-newtag) on how to create a tag.
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
 A person can have any number of tags (including 0)
 </div>
 
 Examples:
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+* `add n/John Doe p/98765432 e/johnd@example.com t/Groom's Friend`
+* `add n/Betsy Crowe t/Bride's Family e/betsycrowe@example.com p/12345678`
```
Comment (kvernnn) [inline docs/UserGuide.md:93]: Should we change Bride's Family to bride's side for more consistent tags in our examples?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -76,32 +80,64 @@ Format: `help`
...
 
+### Creating a new tag: `newtag`
+
+Creates a new tag.
+
+Format: `newtag TAG_NAME`
+* Tag name cannot be more than 50 characters long.
+
+Examples:
+* `newtag Bride's Family`
```
Comment (kvernnn) [inline docs/UserGuide.md:109]: Similarly for here, I think the tags should be consistent. We can use bride's side and groom's side

**Review by kvernnn — COMMENTED** [review]
Minor nits in naming of tags in the examples, otherwise LGTM

**Comment by zeonchew0324** [comment]
lgtm

**Pull request merged**